dev/com.ibm.ws.security.wim.adapter.ldap/src/com/ibm/ws/security/wim/adapter/ldap/LdapConfigManager.java

```diff
@@ -866,7 +866,9 @@ private void setFilters(Map<String, Object> configProps) {
                     List<String> objClses = ldapEntity.getObjectClasses();
                     for (int j = 0; j < objClses.size(); j++) {
                         String objCls = objClses.get(j);
-                        objectClasses.add(objCls);
```

Comment:
I don't believe this change should be here. We only want to make sure the objectclass "*" doesn't end backup on the group entity.